src/components/Admin/AddQuestion.js

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { compose } from "recompose";
+
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+
+import { withFirebase } from "upe-react-components";
+
+import { isLoggedIn } from "../../util/conditions";
+import Loader from "../Loader";
+import Logo from "../Logo";
```

Comment:
Also unused

Reply:
Resolved in commit 15bc7feade4e9e37cdcfb0f3f5de88ed6c4b5193